docs/docs/getting-started/first-steps-local.md

```diff
@@ -4,19 +4,25 @@
 With the `constellation mini` command, you can deploy and test Constellation locally without a cloud subscription. This mode is called MiniConstellation. Conceptually, MiniConstellation is similar to [MicroK8s](https://microk8s.io/), [K3s](https://k3s.io/), and [minikube](https://minikube.sigs.k8s.io/docs/).
 <!-- vale on -->
 
-MiniConstellation uses virtualization to create a local cluster with one control-plane node and one worker node. 
+MiniConstellation uses virtualization to create a local cluster with one control-plane node and one worker node.
 
 :::info
 
-MiniConstellation **doesn't** require hardware with Confidential VM (CVM) support. 
+MiniConstellation **doesn't** require hardware with Confidential VM (CVM) support.
 For attestation, MiniConstellation currently uses the software-based vTPM provided by KVM/QEMU.
 
 :::
 
+:::warning
+
+MiniConstellation has specific soft- and hardware requirements such as a linux OS running on an x86-64 CPU. Please pay close attention to all of the [Prerequisites](#prerequisites) when setting up.
```

Comment:
```suggestion
MiniConstellation has specific soft- and hardware requirements such as a Linux OS running on an x86-64 CPU. Pay attention to all [prerequisites](#prerequisites) when setting up.
```
[prune every excess word](https://learn.microsoft.com/en-us/style-guide/top-10-tips-style-voice#be-brief)